Datagrid/SimplePager.php

```diff
@@ -13,6 +13,9 @@
 
 use Doctrine\Common\Collections\ArrayCollection;
 
+/**
+ * @deprecated Will be removed in version 2.0. Use Sonata\AdminBundle\Datagrid\SimplePager instead.
+ */
```

Comment:
`NEXT_MAJOR` instructions are documented [here](https://github.com/sonata-project/SonataDoctrinePhpcrAdminBundle/blob/1.x/CONTRIBUTING.md#the-base-branch)